packages/cli/src/environments/variables/variables.controller.ee.ts

```diff
@@ -46,27 +54,32 @@ EEVariablesController.post(
 			}
 			throw error;
 		}
-	}),
-);
+	}
+
+	@Get('/:id(\\w+)')
```

Comment:
since variables use our nanoid setup, I think we can remove this regexp now. the default parameter matching will work just fine for variable ids.

```suggestion
	@Get('/:id')
```